language/gpt3/megatron/download_cnndm.py

```diff
@@ -0,0 +1,57 @@
+# experiment config
+dataset_id = "cnn_dailymail"
+dataset_config = "3.0.0"
+text_column = "article"
+summary_column = "highlights"
+
+from datasets import load_dataset
+
+import numpy as np
+import os
+import simplejson as json
+import sys
+
+save_dataset_path = os.environ.get("DATASET_CNNDM_PATH", "data")
+
+# Check whether the specified path exists or not
+isExist = os.path.exists(save_dataset_path)
+if not isExist:
+    # Create a new directory because it does not exist
+    os.makedirs(save_dataset_path)
+
+# Load dataset from the hub
+dataset = load_dataset(dataset_id, name=dataset_config)
+
+instruction_template = "Summarize the following news article:"
```

Comment:
I think this should be `"summarize: "` according to what Google shared.
@morgandu can confirm